.ci/pytorch/win-build.sh

```diff
@@ -30,23 +30,67 @@ if [[ -n "$PYTORCH_FINAL_PACKAGE_DIR" ]]; then
     mkdir -p "$PYTORCH_FINAL_PACKAGE_DIR" || true
 fi
 
-# This directory is used only to hold "pytorch_env_restore.bat", called via "setup_pytorch_env.bat"
+# This directory is used only to hold "pytorch_env_restore.bat", called via "setup_pytorch_env.py"
 CI_SCRIPTS_DIR=$TMP_DIR/ci_scripts
 mkdir -p "$CI_SCRIPTS_DIR"
 
 if [ -n "$(ls "$CI_SCRIPTS_DIR"/*)" ]; then
     rm "$CI_SCRIPTS_DIR"/*
 fi
 
-export SCRIPT_HELPERS_DIR=$SCRIPT_PARENT_DIR/win-test-helpers
+export SCRIPT_HELPERS_DIR=${SCRIPT_PARENT_DIR}/win-test-helpers
+
+# These variables are set for the python build
+if [ $DEBUG == "1" ]
+then
+  export BUILD_TYPE="debug"
+else
+  export BUILD_TYPE="release"
+fi
+
+echo $BUILD_TYPE
+
+"$SCRIPT_HELPERS_DIR"/list_path.bat
+
+echo $PATH
+
+export INSTALLER_DIR=${SCRIPT_HELPERS_DIR}"\installation-helpers"
+
+echo $INSTALLER_DIR
+
+<<'###'
```

Comment:
This appears to be commented out. Remove before marking the PR ready for review.